scenario/LiquidationBotScenario.ts

```diff
@@ -337,6 +359,16 @@ for (let i = 0; i < MAX_ASSETS; i++) {
         exp(500, 18),
         // WBTC
         exp(50, 8),
+      ], 
+      'usdc.n': [
+        // ARB
+        exp(10000, 18),
```

Comment:
Same question here as above. Why lower?

Reply:
Bumped back to the same as original usdc, the new liquidity pool and handle such volume flash loan 👍